<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="fontButton">
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backgroundButton">
      <property name="text">
       <string>Background</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="numberingSectionButton">
      <property name="text">
       <string>Numbering section</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="highlightedTextButton">
      <property name="text">
       <string>Highlighted text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
